### ISSUE TITLE
pre-trained models

### ISSUE THREAD

**Issue by SaharR1372** [thohemp/6DRepNet#22]
Hi,
Thanks for this amazing work. I am really interested in your work. I just want to test your network on the 2 test dataset (AFLW2000 and BIWI). 
I am wondering that why you provide two .pth files (6DRepNet_300W_LP_AFLW2000.pth and 6DRepNet_300W_LP_BIWI.pth ) for each specific test data, should not we just test the network with one pretrained model for both test datasets?

I am looking forward to your response,
Thanks

**Comment by thohemp** [comment]
Hi,
we made multiple runs with the exact same setup and chose the model with the best results for each dataset separately. This highlights what accuracy can be achieved for each test set. In the end, the results only differ marginally. E.g: testing 6DRepNet_300W_LP_BIWI.pth on AFLW2000 gives us:
Yaw: 3.8616, Pitch: 5.0512, Roll: 3.5041, MAE: 4.1390

**Comment by SaharR1372** [comment]
Great. Thanks for your quick response.
